public/create-a-set.html

```diff
@@ -0,0 +1,91 @@
+<!-- Quick education beta testing for firebase-->
+<!DOCTYPE html>
+<html>
+
+<head>
+    <meta charset=utf-8 />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Email/Password Authentication Example</title>
+    <!-- Bootstrap -->
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.1.3/css/bootstrap.min.css"
+        integrity="sha384-MCw98/SFnGE8fJT3GXwEOngsV7Zt27NXFoaoApmYm81iuXoPkFOJwJ8ERdknLPMO" crossorigin="anonymous">
+
+    <link rel="stylesheet" href="css/problem-set.css">
+    <script src="js/create-a-set.js" type="text/javascript"></script>
+</head>
+
+<body>
+    <main class="container">
+        <h2>Create A Problem Set</h2>
+        <form id="set-form">
+            <div class="form-row form-group">
+                <div class="col">
+                    <input type="text" id="title" class="form-control" placeholder="Problem Set Title" rows="2" maxlength="60"
+                        accept-charset="UTF-8">
+                </div>
+            </div>
+
+            <div class="d-flex justify-content-around">
+                <label>Questions:</label>
+                <label>Answers:</label>
+            </div>
+            <div class="form-row form-group">
+                <label for="first-question" class="col-form-label">1.)</label>
+                <div class="col">
+
+                    <input type="text" id="first-question" class="form-control" placeholder="Question" rows="2"
+                        maxlength="60" accept-charset="UTF-8">
+                </div>
+                <div class="col">
+
+                    <input type="text" id="first-answer" class="form-control" placeholder="Answer" rows="2"
+                        maxlength="60" accept-charset="UTF-8">
+                </div>
+            </div>
+            <div class="form-row form-group">
+                <label class="col-form-label">2.)</label>
+                <div class="col">
+                    <input type="text" class="form-control" placeholder="Question" rows="2" maxlength="60"
+                        accept-charset="UTF-8">
+                </div>
+                <div class="col">
+                    <input type="text" class="form-control" placeholder="Answer" rows="2" maxlength="60"
+                        accept-charset="UTF-8">
+                </div>
+            </div>
+            <div class="form-row form-group">
+                <label for="colFormLabel" class="col-form-label">3.)</label>
+                <div class="col">
+                    <input type="text" class="form-control" placeholder="Question" rows="2" maxlength="60"
+                        accept-charset="UTF-8">
+                </div>
+                <div class="col">
+                    <input type="text" class="form-control" placeholder="Answer" rows="2" maxlength="60"
+                        accept-charset="UTF-8">
```

Comment:
Not sure if "accept-charset="UTF-8"" works on input tags since they seem to only be intended for form tags.